Update diagnostics-role-service.md (#61655)
Azure/azure-content: articles/virtual-desktop/diagnostics-role-service.md

## Common error scenarios

> The WVDErrors table tracks errors across all activity types that are tracked by the service. The column ServiceError provides an additional flag that helps you to understand whether the issue is or isn't related to the service. If the column states:

Reviewer: ```suggestion
The WVDErrors table tracks errors across all activity types. The column called "ServiceError" provides an additional flag marked either "True" or "False." This flag will tell you whether the error is related to the service.
```

> * True: this issue can't be mitigated on your side and recommendation is to raise a support ticket under Windows Virtual desktop if you notice an unusual high occorunce. 

Reviewer: ```suggestion
- If the value is "True," then the issue isn't related to the service. In this case, we recommend you submit a support request for Windows Virtual Desktop, especially if you see the "True" flag an unusually high number of times.
```

Reviewer: Can we try this?

If the value is "True," this issue needs to be investigated by the service team. If this impacts user experience or appears a high number of times, we recommend you submit a support ticket for Windows Virtual Desktop.

> * False: you will need to investigate further. The error message itself may hint the area you need to investigate next.

Reviewer: ```suggestion
- If the value is "False," then this is an issue related to the service, and is possibly something you can fix yourself. The error message can give you a clue about where to start investigating.
```

Reviewer: If the value is "False," this is a local service issue that you can possibly fix yourself. The error message can give you a clue about where to start.

> The following table lists common errors your admins might run into and can mitigate.

Reviewer: ```suggestion
The following table lists common errors your admins can help you with.
```
